README.md

```diff
@@ -65,10 +65,16 @@ sudo rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch
 sudo yum install gssntlmssp
 ```
 
-- On Ubuntu 14.04
+- On Ubuntu 14.04 and 16.04
 ```
  sudo apt-get install git pkg-config make g++ rpm librpm-dev libpam0g-dev libssl-dev libkrb5-dev
+ 
 ```
+  for NTLM with SPNEGO functionality (including regression tests) you must use updated packages. Until the corrected packages are available in the regular update distribution, you must add the proposed repos into /etc/apt/sources.list.
```

Comment:
Add a blank line after 72 for readability. Also, most of this file uses shorter lines in the README.md file. Please do that here as well for consistency. Emacs can do that automatically with the escape-q command (assuming standard key bindings).